nautobot/extras/tests/test_views.py

```diff
@@ -908,3 +922,370 @@ def setUpTestData(cls):
             "http_method": "POST",
             "http_content_type": "application/json",
         }
+
+
+@override_settings(JOBS_ROOT=THIS_DIRECTORY)
```

Comment:
Now that #869 is merged, would you mind moving the block of Jobs tests up in this file so that they fall between `GraphQLQueriesTestCase` and `ObjectChangeTestCase` to keep the ordering consistent? (Note that I'm not asking you to separate this group of related tests - they can stay grouped together in that new location even though it's not "quite" alphabetical order overall.

Reply:
Done!